msg/MpcStatistics.msg

```diff
@@ -0,0 +1,11 @@
+# This message provides essential diagnostics information
+
+time stamp
+
+uint16 num_iterations
+float32 problem_update_time
+float32 solve_time
+float32 final_cost
+float32 regularization_rate
+float32 step_length
+float32 feasibility
```

Comment:
```suggestion
 float32 dynamic_feasibility
```

This is optional, but we might want to publish different kind of feasibilities.
For instance, Crocoddyl in future will produce eq/ineq constraint feasibilities.
I personally like to have make a distinction as it is more insightful.

So, if you agree with me, we could also include these:
```c++
float32 equality_feasibility
float32 inequality_feasibility
```

Reply:
Thanks, that's a great point - I'll add them as _infeasibility_ as per usual nomenclature. We may want to have "primal_infeasibility" and "dual_infeasibility" for other types of algorithms, but that is not relevant for now.